### PR TITLE
ABHIGYAN-MOHANTA.json file added

### PR DESCRIPTION
**Pull request by ABHIGYAN-MOHANTA**
## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Just added my profile json file to Link free, not much

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4513"></a>

### CODE REVIEW

File: data/ABHIGYAN-MOHANTA.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Abhigyan Mohanta",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "| Software Engineer | Java | Python | WebDev (MERN Stack) | Twilio Field Operator | #100Devs |",
+  "avatar": "https://github.com/ABHIGYAN-MOHANTA.png",
```
Comment (priyankarpal) [inline data/ABHIGYAN-MOHANTA.json:6]: You don't need to add an avatar. It will automatically fetch from GitHub.
```suggestion

```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by Hrishi5111998 — DISMISSED** [review]
looks good now

File: data/ABHIGYAN-MOHANTA.json (hunk 1 above)
Comment (ABHIGYAN-MOHANTA) [reply data/ABHIGYAN-MOHANTA.json:6]: Oh, thanks! Didn't know about that.

**Review by priyankarpal — APPROVED** [review]
Looks great 👍🏻 Thanks for your contribution. 🥂

**Review by priyankarpal — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**